server/src/main/java/org/apache/druid/indexing/overlord/supervisor/autoscaler/LagStats.java

```diff
@@ -24,12 +24,19 @@ public class LagStats
   private final long maxLag;
   private final long totalLag;
   private final long avgLag;
+  private final ScalingMetric preferredScalingMetric;
```

Comment:
Nit: Thinking about the name again, we could call this field `preferredAggregateForScaling` or even simply `aggregateForScaling`.